apps/server/src/infra/column-client/column-client.adapter.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+import { BoardColumnApi, CardResponse, CreateCardBodyParams, RenameBodyParams } from './generated';
+
+@Injectable()
+export class ColumnClientAdapter {
+	constructor(private readonly columnApi: BoardColumnApi) {}
+
+	public async updateBoardColumnTitle(columnId: string, params: RenameBodyParams): Promise<void> {
+		await this.columnApi.columnControllerUpdateColumnTitle(columnId, params);
+	}
+
+	public async createCard(columnId: string, cardParams: CreateCardBodyParams): Promise<CardResponse> {
+		const cardResponse = (await this.columnApi.columnControllerCreateCard(columnId, cardParams)).data;

```

Comment:
```suggestion
		const { data: cardResponse } = await this.columnApi.columnControllerCreateCard(columnId, cardParams);
```
Logically, destructuring does exactly the same thing. However, the syntax is clearer and more readable. Maintainability is also improved, as additional properties can be easily extended.